src/databricksbundle/detector.py

```diff
@@ -9,3 +11,25 @@ def is_databricks():
     :return: true, if running on databricks
     """
     return os.getenv("DATABRICKS_RUNTIME_VERSION") is not None
+
+
+def is_jupyter_server_running():
+    """
+    Check, if the code is executed by the Jupyter lab instance or locally.
+    :return: true, if running on Jupyter server
+    """
+    return "ipykernel" in sys.modules
+
+
+def jupyter_cwd_setter():
+    """
+    List currently running Jupyter servers and set the current working directory to the root directory of project running the notebook.
+    :return: updated cwd to project root
+    """
+    if is_jupyter_server_running():
```

Comment:
let's put this condition outside this function. The package_config_reader is the best place IMHO.